egs2/TEMPLATE/svs1/db.sh

```diff
@@ -0,0 +1,293 @@
+# Set the path of your corpus
```

Comment:
Maybe we can combine it with the asr's? (then we can just use a sym-link)

Reply:
Individual might be a good choice though, since we are somewhat different, and it can keep the file simpler. We can have some discussion later.